packages/mds-transactions-api/handlers/get-transaction.ts

```diff
@@ -0,0 +1,26 @@
+import { TransactionServiceClient, TransactionDomainModel } from '@mds-core/mds-transactions-service'
+import { isError } from '@mds-core/mds-service-helpers'
+import { ApiRequestParams } from '@mds-core/mds-api-server'
+import { NotFoundError, ServerError } from '@mds-core/mds-utils'
+import { TransactionApiResponse, TransactionApiRequest } from '../@types'
+
+export type TransactionApiGetTransactionRequest = TransactionApiRequest & ApiRequestParams<'name'>
+
+export type TransactionApiGetTransactionResponse = TransactionApiResponse<{ transaction: TransactionDomainModel }>
+
+export const GetTransactionHandler = async (
+  req: TransactionApiGetTransactionRequest,
+  res: TransactionApiGetTransactionResponse
+) => {
+  try {
+    const { name } = req.params
+    const transaction = await TransactionServiceClient.getTransaction(name)
```

Comment:
what is `name` here? I think I'm missing something. Should it be `id`?

Reply:
turd leftover from the generator